From 0da9cf2229de4dd702f38e1a29f1bd73267f7435 Mon Sep 17 00:00:00 2001
From: Kevin Brace <kevinbrace@gmx.com>
Date: Sun, 10 Jul 2016 15:06:10 -0500
Subject: [PATCH 21/21] Version bumped to 0.5.107

This version fixes a bug that causes Sylvania g netbook (VIA
Technologies NanoBook reference design equivalent) to lose VGA
(via DVI to VGA converter) and flat panel when resuming from ACPI
S3 State. Both of them are now working correctly. That being said,
DVI of Sylvania g netbook still gets lost when resuming from ACPI
S3 State and DVI and flat panel still do not work simultaneously.

Signed-off-by: Kevin Brace <kevinbrace@gmx.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 1ac17e8..1eaf903 100644
--- a/configure.ac
+++ b/configure.ac
@@ -23,7 +23,7 @@
 # Initialize Autoconf
 AC_PREREQ(2.57)
 AC_INIT([xf86-video-openchrome],
-        [0.5.106],
+        [0.5.107],
         [https://bugs.freedesktop.org/enter_bug.cgi?product=xorg&component=Driver/openchrome],
         [xf86-video-openchrome])
 
-- 
1.9.1

